parameter/ParameterMgr.h

```diff
@@ -104,11 +103,28 @@ class CParameterMgr
       */
     bool load(std::string& strError);
 
-    // Selection Criteria
-    CSelectionCriterionType* createSelectionCriterionType(bool bIsInclusive);
-    CSelectionCriterion* createSelectionCriterion(const std::string& strName, const CSelectionCriterionType* pSelectionCriterionType);
+    /** Create a new Exclusive criterion
+     *
+     * @param[in] name, the criterion name
+     * @return raw pointer on the created criterion
+     */
+    core::selection::criterion::Criterion*
```

Comment:
Are you sure the client should receive a Criterion object ? Shouldn't it receive a CriterionInterface object instead.